resources/assets/less/bem/beatmapset-header.less

```diff
@@ -39,6 +39,10 @@
 
     padding: (@spacing * 2) @spacing (@spacing * 5 / 2);
 
+    &--main {
+      margin-bottom: auto;
```

Comment:
Looks more like `align-self: stretch` for this and `flex: 1` for `&__beatmap-picker-box`.